packages/tailwindcss-language-service/src/util/find.ts

```diff
@@ -97,9 +97,10 @@ export function findClassListsInCssRange(
   state: State,
   doc: TextDocument,
   range?: Range,
+  lang?: string,
 ): DocumentClassList[] {
   const text = getTextWithoutComments(doc, 'css', range)
-  let regex = isSemicolonlessCssLanguage(doc.languageId, state.editor?.userLanguages)
+  let regex = isSemicolonlessCssLanguage(lang ?? doc.languageId, state.editor?.userLanguages)
```

Comment:
Just a question, in what scenario would `lang` and `doc.languageId` be different?

is that when you have `<style lang="sass">` in an html or vue file where `lang` will be `sass` and `doc.languageId` would be `html`?

Reply:
vue file with a <style> block or a <script> block. Also HTML file with a <script> block. `doc` always refers to the whole file/document rather than just a portion of the file.

Reply:
Gotcha yep that's what I meant and thought. Thanks for verifying!